statsmodels/regression/linear_model.py

```diff
@@ -845,13 +852,42 @@ def fit_regularized(self, method="elastic_net", alpha=0.,
 
         return fit_elasticnet(self, method=method,
                               alpha=alpha,
+                              L1_wt=L1_wt,
                               start_params=start_params,
                               loglike_kwds=loglike_kwds,
                               score_kwds=score_kwds,
                               hess_kwds=hess_kwds,
                               refit=refit,
+                              check_step=False,
                               **defaults)
 
+    def fit_ridge(self, alpha):
```

Comment:
I think this should be privatized for now `_fit_ridge`
Otherwise, we would have to fix the cov_params for it.